tensorflow/core/kernels/conv_ops_gpu_3.cu.cc

```diff
@@ -490,14 +495,63 @@ constexpr bool TileSizeOnFrontier(int TileLongSide, int TileShortSide,
   return TileSizePossibilityFrontierCheck(TileLongSide, TileShortSide,
                                           size_of_t, EqualTo());
 }
-constexpr bool TileSizePastFrontier(int TileLongSide, int TileShortSide,
-                                    int size_of_t) {
+constexpr bool TileSizeOutsideBoundary(int TileLongSide, int TileShortSide,
+                                       int size_of_t) {
   return TileSizePossibilityFrontierCheck(TileLongSide, TileShortSide,
                                           size_of_t, GreaterThan());
 }
+constexpr bool TileSizeOnBoundaryNotFrontier(int TileLongSide,
```

Comment:
Do you mean the boundary of the performant/performance subspace?  If so I don't think this is clear -- in the definitions above, we define this subspace, but we never use the word "boundary", so here it's not clear to me what "boundary" refers to.

Maybe we can fix this as suggested above by removing the concept of this new subspace.

Reply:
I believe I used the terms `boundary` and `interior` according to their topological definitions to save comments; but your way of phrasing it could indeed save even more.